CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Changed
 
+- Breaking change: rename `Call` to `Call16` and `Call17` to `Call` (#125)
```

Comment:
*(I don't know, whether current patch version is ready for review: sorry if the comment is premature.)*

I would not call this change breaking: if a user does not set the build tag, the behaviour is exatly same as before. We just added `Call_16` method.

Reply:
Updated:

```diff
-- Breaking change: rename `Call` to `Call16` and `Call17` to `Call` (#125)
+- Add `Call16` method, support build tag `go_tarantool_call_17`
+  to choose behavior for `Call` method (#125)
```